week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,7 +5,17 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
+  let newArr=[];
+  for (i in strArr){
+    let x = strArr[i].trim();
+    x = x.replace("/","");
+    x = x.toLowerCase();
+    newArr.push(x);
```

Comment:
A big point for the functional approach that you create a new array as an output rather than modifying the input array, it makes it a  function without having no side effect.